engine/apps/api/serializers/channel_filter.py

```diff
@@ -232,5 +196,5 @@ class ChannelFilterUpdateResponseSerializer(ChannelFilterUpdateSerializer):
 
     slack_channel = serializers.SerializerMethodField()
 
-    def get_slack_channel(self, obj) -> SlackChannelDetails | None:
-        return self._get_slack_channel(obj)
+    def _get_slack_channel(self, obj) -> SlackChannelDetails | None:
+        pass
```

Comment:
this is only used in openapi schema "hints", I don't think this function _actually_ needs to do anything (this was the last reference to `_get_slack_channel`)

Reply:
I _think_ we can remove this? `slack_channel` seems to be defined on the parent class as `SlackChannelSerializer` so drf-spectacular should be able to introspect OpenAPI types automatically?

Reply:
good call, thanks for pointing this out! Just removed `ChannelFilterUpdateResponseSerializer` and double-checked swagger spec for `/channel_filters`, looks good 👍 
![Screenshot 2024-11-04 at 1 13 09 PM](https://github.com/user-attachments/assets/d93c0149-cf0c-4216-85f6-42feec51391a)
![Screenshot 2024-11-04 at 1 13 31 PM](https://github.com/user-attachments/assets/317be82d-8564-48fb-b78f-20ca0d2958fe)